index.bs

```diff
@@ -5858,12 +5879,16 @@ Note: In order to interoperate, user agents storing large blobs on authenticator
     };
 
     dictionary AuthenticationExtensionsLargeBlobOutputs {
+        boolean supported;
         ArrayBuffer blob;
         boolean written;
     };
     </xmp>
 
     <div dfn-type="dict-member" dfn-for="AuthenticationExtensionsLargeBlobOutputs">
+        :   <dfn>supported</dfn>
+        ::  [TRUE] if, and only if, the created credential supports storing large blobs. Only present after [=registration extension|registration=].
```

Comment:
>Only present after registration.

I first read this as "present in ceremonies subsequent to a registration ceremony", which is clearly incorrect if you stop to think about it, but perhaps it could be a wee bit clearer. Perhaps something like this?

```suggestion
        ::  [TRUE] if, and only if, the created credential supports storing large blobs. Only present in [=registration extension|registration=] outputs.
```